operator-deployment/minio-operator.yaml

```diff
@@ -158,12 +158,6 @@ rules:
   verbs:
   - "*"
 ---
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: minio-operator-sa
-  namespace: default
----
 kind: ClusterRoleBinding
```

Comment:
we can split all these individual resources into individual files now and kustomize will tail them together

Reply:
done